src/sec_certs/br1/table_parsing/model/entry_types/algorithms.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+
+# Tables 5,6,7,8,9
+# AlgoProp is actually a Key:Value pair, but for simplicity it is represented as a string
+
+
+@dataclass
+class ApprovedAlgo:
```

Comment:
All of these DataClasses are quite verbose, though I like that it allows the type information to carry a lot of the structure and meta information about the data. Do you have any additional descriptions of these fields from the BR1? If so, could you (perhaps in an automated way or not) add them as docstrings to the classes and attributes? Just a single sentence (if available) would suffice.

The names are usually quite descriptive so this is not that important, more of a nice to have.